etc/config.sample.toml

```diff
@@ -330,6 +330,7 @@ reporting-disabled = false
   # batch-pending = 5 # number of batches that may be pending in memory
   # batch-timeout = "1s" # will flush at least this often even if we haven't hit buffer limit
   # read-buffer = 0 # UDP Read buffer size, 0 means OS default. UDP listener will fail if set above OS max.
+  # udp-payload-size = 1500 # set the expected UDP payload size, default: 65536
```

Comment:
Also, our convention is that we set the value of commented out flags to the default. However, a note here explaining that a lower value would be better for most would make sense.

Sound good?